modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/criteria/contributor/test/UserSegmentsCriteriaContributorTest.java

```diff
@@ -225,47 +230,36 @@ public void testGetFieldsWithOptions() throws Exception {
 		SegmentsCriteriaContributor segmentsCriteriaContributor =
 			_getSegmentsCriteriaContributor();
 
-		List<Field> fields = segmentsCriteriaContributor.getFields(
-			_getMockPortletRequest());
-
-		Stream<Field> fieldsStream = fields.stream();
-
-		Optional<Field> fieldOptional = fieldsStream.filter(
+		List<Field> filteredFields = ListUtil.filter(
+			segmentsCriteriaContributor.getFields(_getMockPortletRequest()),
 			field -> StringUtil.endsWith(
 				field.getName(),
-				Normalizer.normalizeIdentifier(expandoColumn.getName()))
-		).findFirst();
+				Normalizer.normalizeIdentifier(expandoColumn.getName())));
+
+		Optional<Field> fieldOptional = Optional.ofNullable(
+			filteredFields.get(0));
```

Comment:
@jeffwu0724 Although it's a test, we don't want to always go through the whole list when we can shortcut earlier. Please make this and the change below a for-loop with shortcut.